base/threadingconstructs.jl

```diff
@@ -93,31 +97,50 @@ function _threadsfor(iter,lbody)
 end
 
 """
-    Threads.@threads
+    Threads.@threads [schedule] for ... end
 
-A macro to parallelize a for-loop to run with multiple threads. This spawns [`nthreads()`](@ref)
-number of threads, splits the iteration space amongst them, and iterates in parallel.
-A barrier is placed at the end of the loop which waits for all the threads to finish
-execution, and the loop returns.
+A macro to parallelize a `for` loop to run with multiple threads. Splits the iteration
+space among multiple tasks and runs those tasks on threads according to a scheduling
+policy.
+A barrier is placed at the end of the loop which waits for all tasks to finish
+execution.
+
+The `schedule` argument can be used to request a particular scheduling policy.
+The only currently supported value is `:static`, which creates one task per thread
+and divides the iterations equally among them. Specifying `:static` is an error
+if used from inside another `@threads` loop or from a thread other than 1.
+
+The default schedule (used when no `schedule` argument is present) is subject to change.
+
+!!! compat "Julia 1.5"
+    The `schedule` argument is available as of Julia 1.5.
```

Comment:
```suggestion
    The `schedule` argument is available as of Julia 1.5.
    Nesting `@threads` calls now throws an exception.
```

Reply:
Only if you specify `:static`; I fixed that in the doc string.